### PR TITLE
Add WFS Sorting notes

### PR DESCRIPTION
**Pull request by geographika**
Add notes on which drivers allow WFS sorting. 
Related to https://github.com/mapserver/mapserver/pull/5833

### CODE REVIEW

File: en/ogc/wfs_server.txt [hunk 1]
```diff
@@ -374,6 +374,26 @@ WFS GetFeature requests. For more information on the server side of
...
 <filter_encoding>`.
 
+Support for Sorting
+*******************
+
+Sorting is supported for WFS 1.1 and 2.0. WFS clients can add a ``SORTBY`` parameter
+to the query string to order results based on one or more attribute values 
+e.g. ``SORTBY=Field1 DESC,Field2 ASC``. 
+
+The list of drivers support sorting, along with the version of MapServer they were added
```
Comment (havatv) [inline en/ogc/wfs_server.txt]:
```suggestion
The list of drivers that support sorting, along with the version of MapServer they were added:
```

File: en/ogc/wfs_server.txt [hunk 2]
```diff
@@ -374,6 +374,26 @@ WFS GetFeature requests. For more information on the server side of
...
+is below:
```
Comment (havatv) [inline en/ogc/wfs_server.txt]:
```suggestion
```

File: en/ogc/wfs_server.txt [hunk 3]
```diff
@@ -374,6 +374,26 @@ WFS GetFeature requests. For more information on the server side of
...
+
+- OGR (7.0+)
+- PostGIS (7.0+)
+- Oracle (7.0.7+)
+- MS SQL Server (7.4.1+)
+
+The MapServer shapefile provider does not support sorting. If this needed, you must
```
Comment (havatv) [inline en/ogc/wfs_server.txt]:
```suggestion
The MapServer Shapefile provider does not support sorting. If this needed, you must
```

**Comment by havatv** [comment]
Is it worth backporting some of this to earlier versions (support for WFS sorting seems to have been introduced in 7.0).

**Comment by geographika** [comment]
@havatv - just checked the source for the 7.0 branch and Oracle was backported - https://github.com/mapserver/mapserver/blob/branch-7-0/maplayer.c#L1568
Will update the list.

File: en/ogc/wfs_server.txt [hunk 4]
```diff
@@ -374,6 +374,26 @@ WFS GetFeature requests. For more information on the server side of
...
 <filter_encoding>`.
 
+Support for Sorting
+*******************
+
+Sorting is supported for WFS 1.1 and 2.0. WFS clients can add a ``SORTBY`` parameter
+to the query string to order results based on one or more attribute values 
+e.g. ``SORTBY=Field1 DESC,Field2 ASC``. 
+
+The list of drivers that support sorting is below. Sorting was implemented in
```
Comment (havatv) [inline en/ogc/wfs_server.txt]:
```suggestion
The list of drivers that support sorting (sorting was implemented in 
```

File: en/ogc/wfs_server.txt [hunk 5]
```diff
@@ -374,6 +374,26 @@ WFS GetFeature requests. For more information on the server side of
...
 
+Support for Sorting
+*******************
+
+Sorting is supported for WFS 1.1 and 2.0. WFS clients can add a ``SORTBY`` parameter
+to the query string to order results based on one or more attribute values 
+e.g. ``SORTBY=Field1 DESC,Field2 ASC``. 
+
+The list of drivers that support sorting is below. Sorting was implemented in 
+MapServer 7.0 unless otherwise specified:
```
Comment (havatv) [inline en/ogc/wfs_server.txt]:
```suggestion
MapServer 7.0 unless otherwise specified):
```

**Comment by havatv** [comment]
Backporting: Sorry if I was unclear - what I meant backporting to QGIS 7.0 (and 7.2) **documentation**.

**Pull request merged**